src/stable/iomesh/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+appVersion: v5.3.0-rc13
+dependencies:
+- name: zookeeper-operator
+  repository: http://iomesh.com/charts
+  version: 0.2.15
+- name: hostpath-provisioner
+  repository: http://iomesh.com/charts
+  version: 0.5.3
+- condition: openebs-ndm.enabled
+  name: openebs-ndm
+  repository: http://iomesh.com/charts
+  version: 1.8.0
+- name: csi-driver
+  repository: http://iomesh.com/charts
+  version: 2.6.0
+- name: blockdevice-monitor
```

Comment:
This chart is not exists. https://github.com/kubesphere/helm-charts/actions/runs/8601181645/job/23567821533?pr=386#step:3:331

I'll revert this PR